code/modules/cargo/blackmarket/blackmarket_items/weapons.dm

```diff
@@ -60,4 +70,142 @@
+
+	price_min = 100
+	price_max = 600
+	stock_max = 6
+	availability_prob = 40
+
+/datum/blackmarket_item/weapon/himehabu_mag
+	name = "Himehabu Magazines"
+	desc = "Compact .22lr magazines for use in the Himehabu pistol."
+	item = /obj/item/ammo_box/magazine/m22lr
+
+	price_min = 100
+	price_max = 200
+	stock_max = 6
+	availability_prob = 0
+
+/datum/blackmarket_item/weapon/e10
+	name = "E-10 Laser Pistol"
+	desc = "Sharplite letting you down? Try these classic Eoehoma Firearms E-10 Laser Pistols."
+	item = /obj/item/gun/energy/laser/e10
+
+	price_min = 500
+	price_max = 1250
+	stock_max = 5
+	availability_prob = 20
+
+/datum/blackmarket_item/weapon/e11
+	name = "E-11 Energy Gun"
+	desc = "Look. I'll be straight with you. These guns are awful. But, they are cheap if you're that desperate."
+	item = /obj/item/gun/energy/e_gun/e11
+
+	price_min = 250
+	price_max = 750
+	stock = 5
+	availability_prob = 60
+
+/datum/blackmarket_item/weapon/e40
+	name = "E-40 Hybrid Assault Rifle"
+	desc = "A dual mode hybrid assault rifle made by the now defunct Eoehoma Firearms. Capable of firing both bullets AND lasers, for the discerning dealer in death. Chambered in Eoehoma .299 Caseless."
+	item = /obj/item/gun/ballistic/automatic/assault/e40
+	pair_item = /datum/blackmarket_item/weapon/e40_mag
+
+	price_min = 7000
+	price_max = 13000
+	stock = 1
+	availability_prob = 20
+
+/datum/blackmarket_item/weapon/e40_mag
+	name = "Eoehoma .299 Caseless Magazine"
+	desc = "A 30 round magazine for the E-40 Hybrid Rifle."
+	item = /obj/item/ammo_box/magazine/e40
+
+	price_min = 750
+	price_max = 1250
+	stock_max = 3
+	availability_prob = 0
+
+/datum/blackmarket_item/weapon/e50
+	name = "E-50 Energy Emitter"
+	desc = "An Eoehoma Firearms E-50 Emitter cannon. For when you want a send a message. A really big message."
+	item = /obj/item/gun/energy/laser/e50
+
+	price_min = 4000
+	price_max = 7000
+	stock_max = 2
+	availability_prob = 20
+
+/datum/blackmarket_item/weapon/saber_smg
+	name = "Saber 9mm SMG"
+	desc = "A prototype 9mm submachine gun. Most of these never got past the RND phase and into distribution. But we happen know a guy."
+	item = /obj/item/gun/ballistic/automatic/smg/proto
+	pair_item = /datum/blackmarket_item/weapon/saber_mag
+
+	price_min = 2500
+	price_max = 4200
+	stock_max = 2
+	availability_prob = 25
+
+/datum/blackmarket_item/weapon/saber_mag
+	name = "Saber 9mm SMG Magazines"
+	desc = "Magazines for use in the Saber 9mm SMG. No, they don't work as swords."
+	item = /obj/item/ammo_box/magazine/smgm9mm
+
+	price_min = 500
+	price_max = 1000
+	stock_max = 2
+	availability_prob = 0
+
+/datum/blackmarket_item/weapon/bg_16
+	name = "BG-16 Beam Gun"
+	desc = "Not satisfied by Etherbor's civilian offerings? Try this military grade one we found!"
+	item = /obj/item/gun/energy/kalix/pgf
+
+	price_min = 2500
+	price_max = 5000
+	stock = 2
+	availability_prob = 20
+
+/datum/blackmarket_item/weapon/bioterror_foam
```

Comment:
isn't this nukie gear? this seems like a really, really bad idea

Reply:
Removed as requested.